app_test.py

```diff
@@ -121,4 +121,5 @@ def user_input_images(query):
 if __name__ == "__main__":
```

Comment:
High severity: The entire main guard block lacks indentation as required by Python syntax, which will cause a runtime IndentationError and prevent execution. Fix indentation to nest the if-blocks correctly under the main guard. Score: 6/10. Suggest this fix:
```suggestion
if __name__ == "__main__":
    if query_text_input != '':
        user_input(query_text_input)
        user_input_images(query_text_input)
        print("Query Text Input:", query_text_input)
```